src/components/CookiesToaster/CookiesToaster.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useEffect, useState } from "react";
+import CookiesImage from "@/src/assets/WEBP/CookiesGirl.webp";
+import styles from "./CookiesToaster.module.scss";
+import { Icon, IconEnum } from "..";
+
+const CookiesToaster: FC = () => {
+  const [toasterHide, setToasterHide] = useState(false);
+
+  useEffect(() => {
+    const acceptedCookies = localStorage.getItem("CookieAccept");
+    return () => {
+      if (acceptedCookies === "1") {
+        setToasterHide(true);
+      }
+    };
+  }, []);
+
+  const CookieAccept = () => {
```

Comment:
name of methods start from loverCase - cookieAccept

Reply:
fixed